src/main/java/com/photolocos/enterprise/dao/PhotoRepository.java

```diff
@@ -1,20 +1,21 @@
 package com.photolocos.enterprise.dao;
 
+import com.photolocos.enterprise.dto.LocationDTO;
 import com.photolocos.enterprise.dto.PhotoDTO;
 import org.springframework.context.annotation.Profile;
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.List;
 
-
+@Profile("!test")
 public interface PhotoRepository extends CrudRepository<PhotoDTO, Integer> {
 
-    List<PhotoDTO> findByTagsContaining(String tags);
+    List<PhotoDTO> findByTagsContaining(String tags) throws Exception;
 
-    List<PhotoDTO> findByRating(int rating);
+    List<PhotoDTO> findByRating(int rating) throws Exception;
 
-    List<PhotoDTO> findByType(String type);
+    List<PhotoDTO> findByType(String type) throws Exception;
 
-    PhotoDTO findByLocation(int location);
+    List<PhotoDTO> findByLocation(LocationDTO location) throws Exception;
 
 }
```

Comment:
Added @Profile("!test") annotation. Adding these annotations to the Repository classes was not enough to prevent spring boot from attempting a connection. This is why I added a profile specific application properties file which stops spring boot from automatically attempting a connection.